sdk/nodejs/proto/resource_pb.js

```diff
@@ -5914,7 +5914,8 @@ proto.pulumirpc.TransformResponse.prototype.hasOptions = function() {
 proto.pulumirpc.Result = {
   SUCCESS: 0,
   FAIL: 1,
-  SKIP: 2
+  SKIP: 2,
+  FAILED_STEP_GENERATION: 3
```

Comment:
Does this actually need to be a new code? Isn't it just a FAIL as far as users care.